make_queue/migrations/0010_auto_20190302_2210.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.5 on 2019-03-02 21:10
+
+from django.db import migrations
+import make_queue.models.course
```

Comment:
Rename file to `0010_unique_username_in_course_registration.py` to make it easier to see what migrations does (I just recently found out that you could rename the migrations, and think it makes sense to do this for all future migrations).

Reply:
Yes, that is useful.